tests/Performance/LazyLoading/ProxyInitializationTimeBench.php

```diff
@@ -27,45 +29,57 @@ final class ProxyInitializationTimeBench
 
     public function init(): void
     {
-        $proxyFactory = (new NonProxyLoadingEntityManager(EntityManagerFactory::getEntityManager([])))
+        $proxyFactory = (new NonProxyLoadingEntityManager($em = EntityManagerFactory::getEntityManager([])))
             ->getProxyFactory();
 
+        if (! $em->getConfiguration()->isNativeLazyObjectsEnabled()) {
+            throw new LogicException('Native lazy objects should be enabled.');
+        }
+
+        $userReflection     = new ReflectionClass(CmsUser::class);
+        $employeeReflection = new ReflectionClass(CmsEmployee::class);
+
         for ($i = 0; $i < 10000; ++$i) {
             $this->cmsUsers[$i]             = $proxyFactory->getProxy(CmsUser::class, ['id' => $i]);
             $this->cmsEmployees[$i]         = $proxyFactory->getProxy(CmsEmployee::class, ['id' => $i]);
             $this->initializedUsers[$i]     = $proxyFactory->getProxy(CmsUser::class, ['id' => $i]);
             $this->initializedEmployees[$i] = $proxyFactory->getProxy(CmsEmployee::class, ['id' => $i]);
 
-            $this->initializedUsers[$i]->__load();
-            $this->initializedEmployees[$i]->__load();
+            clone $this->initializedUsers[$i];
```

Comment:
using `$em->getUnitOfWork()->initializeObject($entity)` could allow making the bench compatible with both modes, and will also make the intent more explicit.

Reply:
Ah nice! Let me try that 👍

Reply:
It's weird to me that `ReflectionClass::initializeObject` isn't mentioned under https://www.php.net/manual/en/language.oop5.lazy-objects.php